test/constrained-generics/error-call-ug.good

```diff
@@ -0,0 +1,10 @@
+resolving interface declaration 'IFC'  error-call-ug.chpl:9
+  required fn 'reqFun'
+  required fn 'dfltImpl'  with a default implementation
+
+resolving CG function early 'cgFun'  error-call-ug.chpl:16
+error-call-ug.chpl:16: In function 'cgFun':
+error-call-ug.chpl:18: error: unresolved call 'ugFun(Q)'
+error-call-ug.chpl:21: note: this candidate did not match: ugFun(arg)
+error-call-ug.chpl:18: note: because call actual argument #1 of an interface type Q
+error-call-ug.chpl:21: note: is passed to a generic formal
```

Comment:
I think this error message should say something about how you can't call an unconstrained generic from a constrained generic.

Reply:
Maybe just adding `note: cannot call an unconstrained generic function from a constrained generic` or something along those lines as an additional line